ecosystem/join.html

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Join
+permalink: ecosystem/join.html
+body-class: ecosystem
+---
+
+<div class="jumbotron jumbotron-fluid join-jumbotron">
+  <div class="container">
+    <h1>Join The<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span>Pytorch Ecosystem</span></h1>
+
+  </div>
+</div>
+
+<div class="main-content-wrapper join-wrapper">
+  <div class="main-content">
+    <div class="container">
+
+      <!-- START CONTENT -->
+
+      <div class="row">
+        <div class="col-md-12 join-intro">
+          <div class="intro-message">
+            <p>Thank you for being part of the PyTorch community! The PyTorch ecosystem consists of projects, tools, and libraries from a broad set of researchers in academia and industry, application developers, and ML engineers.
+            The goal of this ecosystem is to support, accelerate, and aid in your exploration with PyTorch. Below are a few questions to think about if you'd like to have your project included as part of the ecosystem:</p>
+          </div>
+
+          <div class="row requirements">
+            <div class="col-md-1 join-number">
+              <h1>01</h1>
+            </div>
+
+            <div class="col-md-11">
+              <p><span>Can your project be used with or within PyTorch to help augment the user experience, enable new capabilities or speed up training/inference?</span><br>
+              Examples could include visualization tools, a kernel library or a framework that sits on top to enable research in a particular area such as NLP.</p>
+            </div>
+
+          </div>
+
+          <div class="row requirements">
+            <div class="col-md-1 join-number">
+              <h1>02</h1>
+            </div>
+
+            <div class="col-md-11">
+              <p><span>Is the project ready for broad developer usage?</span><br>
+              For example, is the project stable, will it be maintained, and is there adequate supporting infrastructure, documentation, and technical support to allow a developer to successfully use it?</p>
+            </div>
+          </div>
+
+          <p>If you answered 'yes' to both, please send us an email with more details on the following, and we'll follow up with next steps:</p>
```

Comment:
"please send us an email" is a little confusing to me. I would expect one of two things here? An email-based href to compose an email -or- an explicit email address shown.

Unless you are saying the form below sends the email? If that is the case, I would probably say:

"please fill out the form below with more details, and we'll....."

Reply:
I updated the language. Just let me know if you have any other feedback.